Src/mesg.c

```diff
@@ -38,8 +38,8 @@ static int	sa_snd(Queue *, Lextok *);
 static int	s_snd(Queue *, Lextok *);
 extern Lextok	**find_mtype_list(const char *);
 extern char	*which_mtype(const char *);
-extern void	sr_buf(int, int, const char *);
-extern void	sr_mesg(FILE *, int, int, const char *);
+void	sr_buf(int, int, const char *);
```

Comment:
Indent.

Reply:
in fact, it is indented the way it should be - the convention is that the fun name is separated with tab from its type - here, there is no need for "extern", so, it looks the same as, for instance, decl. in line 29 or 30